app/main/views/g_cloud.py

```diff
@@ -365,7 +365,7 @@ def save_search(framework_framework):
             form.name.errors = ["Names must be between 1 and 100 characters"]
 
         if not save_search_selection and request.method == "POST":
-            save_search_selection_error = "Please choose where to save your search."
+            save_search_selection_error = "Please choose where to save your search"
```

Comment:
Can we refactor this so that the content is in the template? Ideally as much content as possible should be there rather than in code files.